### PR TITLE
Button size cleanup

### PR DESCRIPTION
**Pull request by AdamMitchell-ms**
## Overview
Fixes the in-editor hit target size.  Also clean up the size of a few other things.

## Changes
- Fixes: #6065.
   - Previously, RectTransformCubeScaler was on the same gameobject as the cube.  In response to the RectTransforms size changes, it applied a scale to itself to cause the cube to get resized.  This also caused the RectTransform object itself to get scaled, resulting in the hit target in editor being very large.
   - Now, the cube has a transform (not a RectTransform).  RectTransformCubeScaler is on the parent object.  It listens to the size of that parent object and applies a scale to the child transform where the cube lives.
- Resize the other UnityUI PressableButtons to be 32x32.
- Change the hierarchy of the SceneDescriptionPanel to avoid similar issues of clicks outside of it causing it to be selected.

### CODE REVIEW

**Comment by AdamMitchell-ms** [comment]
Ok, I retargeted, but now I think I need to rebase so I don't bring in other changes.  I haven't done a branch retargeting on github before, so I'm not familiar with best practices.

**Comment by AdamMitchell-ms** [comment]
Per discussion with @julenka, decided to target mrtk_development for now.

File: Assets/MixedRealityToolkit.SDK/Features/UX/Scripts/Utilities/RectTransformCubeScaler.cs [hunk 1]
```diff
@@ -17,6 +17,10 @@ public class RectTransformCubeScaler : MonoBehaviour
         private RectTransform rectTransform;
         private Vector2 prevRectSize = Vector2.zero;
 
+        [SerializeField]
+        private Transform objectToScale = null;
```
Comment (thalbern) [inline Assets/MixedRealityToolkit.SDK/Features/UX/Scripts/Utilities/RectTransformCubeScaler.cs]: if this is serialized and visible in the editor we should document it (+tooltip)

File: Assets/MixedRealityToolkit.SDK/Features/UX/Scripts/Utilities/RectTransformCubeScaler.cs [hunk 2]
```diff
@@ -17,6 +17,10 @@ public class RectTransformCubeScaler : MonoBehaviour
...
+        private Transform prevObjectToScale = null;
+
         private void Start()
         {
             rectTransform = GetComponent<RectTransform>();
```
Comment (thalbern) [inline Assets/MixedRealityToolkit.SDK/Features/UX/Scripts/Utilities/RectTransformCubeScaler.cs]: does it make sense to have the gameobject this component is attached to set up as default if the objectToScale = null?

File: Assets/MixedRealityToolkit.SDK/Features/UX/Scripts/Utilities/RectTransformCubeScaler.cs (hunk 1 above)
Comment (AdamMitchell-ms) [reply Assets/MixedRealityToolkit.SDK/Features/UX/Scripts/Utilities/RectTransformCubeScaler.cs]: Good point.  I'll add a comment & tooltip here.

File: Assets/MixedRealityToolkit.SDK/Features/UX/Scripts/Utilities/RectTransformCubeScaler.cs (hunk 2 above)
Comment (AdamMitchell-ms) [reply Assets/MixedRealityToolkit.SDK/Features/UX/Scripts/Utilities/RectTransformCubeScaler.cs]: I thought about that, but decided against it.  Doing that is what caused the in-editor clickable region to be so large in the first place, so I don't think it's a good default behavior.

File: Assets/MixedRealityToolkit.SDK/Features/UX/Scripts/Utilities/RectTransformCubeScaler.cs (hunk 1 above)
Comment (AdamMitchell-ms) [reply Assets/MixedRealityToolkit.SDK/Features/UX/Scripts/Utilities/RectTransformCubeScaler.cs]: Added a tooltip.  I think it explains the correct usage.

File: Assets/MixedRealityToolkit.Examples/Demos/HandTracking/Prefabs/PressableButtonsTestCanvas.prefab [hunk 3]
```diff
@@ -1142,7 +1127,7 @@ PrefabInstance:
     - target: {fileID: 3223503683246946309, guid: 2f626628bde0879488068de0e9f25f8d,
         type: 3}
       propertyPath: m_LocalPosition.z
-      value: 0.000029802322
+      value: -0
```
Comment (davidkline-ms) [inline Assets/MixedRealityToolkit.Examples/Demos/HandTracking/Prefabs/PressableButtonsTestCanvas.prefab]: why _minus_ zero?

**Pull request merged**